### PR TITLE
FIX: Fixes old theme file lookup path in `Mautic\CoreBundle\Twig\Helper\ThemeHelper`

### PR DESCRIPTION
**Pull request by putzwasser**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [X]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

This fixes the M4 lookup path for the error page twig templates.

### CODE REVIEW

**Comment by escopecz** [comment]
The code change seem correct. Do you have some steps to test this? The testers must reproduce it and confirm this change is fixing the issue.

**Comment by putzwasser** [comment]
> Do you have some steps to test this?

I thought this was obvious, TBH. Before a template was not loaded, after PR your template gets loaded.

1. Create an error code template in your themes folder:
    ```
	themes
	├── mytheme
	│   ├── assets
	│   │   └── ...
	│   ├── config.json
	│   ├── css
	│   │   └── styles.css
	│   ├── error_404.html.twig <== Create this
	│   ├── html
	│   │   ├── ...html.twig
	│   │   └── ...html.twig
	│   └── thumbnail.png
	└── ...
    ```
2. Navigate to a page that doesn't exist. `mtc.mysite.com/does-not-exist`

**Before PR**
3. See the default Mautic 404 page.

**After PR**
3. See your 404 page.

**Review by escopecz — APPROVED** [review]
Code change is correct. I let someone to follow the steps to manually test this.

**Comment by LordRembo** [comment]
@putzwasser can you rebase your branch to be up to date? Trying to run it as-is, I'm just getting php errors when starting it up.

**Comment by escopecz** [comment]
@LordRembo I merged current 5.x into this PR. Does it solve your issue? If not, can you elaborate what are you trying to do?

**Comment by LordRembo** [comment]
@escopecz that fixes things, thanks. I'm just pulling in this project to test the error page

**Comment by LordRembo** [comment]
Tested it as described but not seeing the replaced template:
- checked which theme is active, in the Settings (it's the 'Blank' one)
- added an empty `error_404.html.twig` with some plain text in it
- tried it in several places: directly in the `themes/blank` folder, and in the `themes/blank/html` folder
- didn't load when surfing to a non-existing page, it just shows the default 404

**Review by LordRembo — CHANGES_REQUESTED** [review]
Code looks good, but either the described testing process is faulty or there is a change missing, because I could not get a new theme file to load for a 404 error.

**Comment by putzwasser** [comment]
> * didn't load when surfing to a non-existing page, it just shows the default 404

Works for me. My best guess: **You are probably currently logged into your backend/dashboard.**

For some reason Mautic will behave differently in many places when you're logged in. Sometimes this makes sense, sometimes it does not (to me at least).

Rendering the default 404 is one of this cases.

**TL;DR: Use an incognito tab and try again. If you can't confirm it works, there must be a bug.**

**Review by LordRembo — APPROVED** [review]

**Comment by LordRembo** [comment]
@putzwasser Alright, tested again, it works.
For other PR's, please approach the description of the testing process from the standpoint of a novice. The more detail, the better.
